README.md

```diff
@@ -5,7 +5,7 @@ Modeling of published clinical Probenecid-Furosemide-DDI studies for model evalu
 Within this repository, we share a PK-Sim project file containing simulations of all published clinical studies used to evaluate the predictive performance of our models regarding the [Probenecid](https://github.com/Open-Systems-Pharmacology/Probenecid-Model)-[Furosemide](https://github.com/Open-Systems-Pharmacology/Furosemide-Model)-DDI, including the respective observed data. For further details and documentation please refer to [[1](#reference)].
```

Comment:
Please change "share PK-Sim project" to "Share PK-Sim snapshot" (or how is this formulated in other DDI repos...)